versioned/storage/jdbc/src/main/java/org/projectnessie/versioned/storage/jdbc/DatabaseSpecifics.java

```diff
@@ -141,4 +141,63 @@ public String wrapInsert(String sql) {
       return sql + " ON CONFLICT DO NOTHING";
     }
   }
+
+  static class MariaDBDatabaseSpecific implements DatabaseSpecific {
```

Comment:
You also have to modify the `detect` method and add 2 more case branches: `"mariadb"` and `"mysql"`.

Reply:
yes i have added in subsequent change